captslog/ui/main.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import absolute_import, unicode_literals
-from PyQt4 import QtCore, QtGui
+from PySide import QtCore, QtGui
```

Comment:
Open `cmd`.
Type `python`
Inside of python running environment, type `from PyQt4 import QtCore`